mender-convert.sh

```diff
@@ -551,31 +543,57 @@ do_mender_disk_image_to_artifact() {
   fi
 
   if [ $ret -eq 0 ]; then
-    local rootfs_file=${output_dir}/rootfs.ext4
-
-    echo "Creating a ext4 file-system image from modified root file-system"
-    dd if=/dev/zero of=$rootfs_file seek=${prootfs_size} count=0 bs=512 status=none
-
-    sudo mkfs.ext4 -FF $rootfs_file -d $rootfs_path
-
-    fsck.ext4 -fp $rootfs_file
```

Comment:
Why have you removed checking of extracted file system?

Reply:
I extract the ext4 root filesystem from Mender image. I would expect it is correct. But ok, I can add it back.